test-cases/CVE-2022-3544.h

```diff
@@ -0,0 +1,319 @@
+// this is the header file for CVE-2022-3544
+//#pragma once
+#include <malloc.h>
+#include <stdbool.h>
+//#include "damon_operations.h"
```

Comment:
How come there are comments here for //#pragma once and //#include "damon_operations.h"? If they are just here for preservation, you can definitely remove them now as it'll be saved on GitHub. If not, add a comment why these include directives are commented out